extensions/docbook-snippets/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "docbook-snippets",
+  "displayName": "DocBook Snippets",
+  "description": "A set of often-used DocBook snippets",
+  "version": "0.0.1",
```

Maybe use a higher version number?
I fear it has a psychological effect that users perceive it as "unstable" or "incomplete". You could have something like "0.1.0" if you think there is something missing or just raise it to "1.0.0".

```suggestion
  "version": "0.1.0",
```